app/lib/actions/katello/content_view/incremental_updates.rb

```diff
@@ -4,8 +4,8 @@ module ContentView
       class IncrementalUpdates < Actions::EntryAction
         include Helpers::Presenter
 
-        def plan(version_environments, composite_version_environments, content, dep_solve, hosts, description,
-                use_remote_execution = false)
+        def plan(version_environments, composite_version_environments, content, dep_solve, hosts, description) # rubocop:disable Metrics/MethodLength
+          use_remote_execution = true # TODO: remove this when we remove katello-agent dynflow actions
```

Comment:
This makes code on line 38-40 would never run. 
Do we really need it?

Reply:
I didn't want to get too deep into editing Dynflow actions here, since Dynflow actions will be in another PR.